### ISSUE TITLE
Docs review and update

### ISSUE THREAD

**Issue by jhkrug** [kubewarden/docs#204]
Keeping track of @jhkrug's docs work. 

The goals?

- Docs as part of community building. How to progress?
- Rancher Prime integration. What's possible?
- Enterprise-ready lighthouse support goal. What's possible beyond good documentation.
- Clear document architecture with process for deciding an information chunks place in the architecture. Clear for personas what to look at.
- Finished first full review, readability, grammar, spelling, style-guide. 'General improvements' to improve useability
- Documentation coverage. Are core topics completely covered? Are topics missing?
- Site styling. Anything? Not that important?

## Architecture

Do we need any structure changes to better match [Diataxis](https://diataxis.fr/)/Divio guidelines? Yes, we do.

### Acceptance criteria

- [x] #152 
- [x] #332
- [x] #348 
- [x] #351
- [x] #354 - Comes out of the work on #351. Needs further attention.

once #332 is complete then it makes sense to finish:

- [x] #325 

## Full review
Questions to be addressed for each of the issues (issues roughly correspond to a review and update of a page or section) below:

- Is the content up to date and accurate.
- Any screenshots reflect the current state of products. 
- Installation scripts and tutorial examples work as listed and produce the documented output. 
- Grammar and style reflects the guidelines (Microsoft & SUSE).

### General

- [x] #206
- [x] #211 
- [x] #212  
- [x] #217 
- [x] #219
- [x] #220
- [x] #221
- [x] #222
- [x] #223 
- [x] #224  
- [x] #226 
- [x] #234  
- [x] #251
- [x] #258 
- [x] #270

### Reference docs

- [x] #248 
- [x] #253 
- [x] #254
- [x] #255
- [x] #256 

### Policy specification

- [x] #359 will close - 01-intro-spec.md
- [x] #359 will close - 02-settings.md
- [x] #359 will close - 03-validating-policies.md
- [x] #360 will close - 04-mutating-policies.md
- [x] #360 will close - 05-context-aware-policies.md
- [x] #361 will close -  host-capabilities 1 - intro
- [x] #361 will close -  host-capabilities 2 - sig verify
- [x] #361 will close -  host-capabilities 3 - container registry
- [x] #361 will close -  host-capabilities 4 - net
- [x] #361 will close -  host-capabilities 5 - crypto
- [x] #361 will close -  host-capabilities 6 - kubernetes

### Operator Manual

- [x] #363 will close this. Metrics reference
- [X] #263 CRDs
- [x] Already done as part of earlier work. Verification Config

### Rancher UI Extension

- [x] #364 closes - Quickstart
- [x] #364 closes - Monitoring
- [x] #364 closes - Tracing

### Air gap

- [x] #369 completes - Requirements
- [x] #369 completes - Installation

### Writing Policies

- [x] #282

#### Rust

#295 closes all these:

- [x] #294 
- [x] 04-write-validation-logic.md
- [x] 05-mutation-policy.md
- [x] 06-logging.md
- [x] 07-build-and-distribute.md

#### Go

- [x] #321 

#### Rego

- [x] PR #368 - 01-intro-rego.md
- [x] PR #368 - 02-builtin-support.md

##### Rego - gatekeeper

- [x] PR #368 - 01-intro.md
- [x] PR #368 - 02-create-policy.md
- [x] PR #368 - 03-build-and-run.md
- [x] PR #368 - 04-distribute.md

##### Rego open-policy-agent

- [x] PR #368 - 01-intro.md
- [x] PR #368 - 02-create-policy.md
- [x] PR #368 - 03-build-and-run.md
- [x] PR #368 - 04-distribute.md

#### WASI

Closed by #314 

- [x] 01-intro-wasi.md
- [x] 02-raw-policies.md

#### Others

All covered in, and will be closed by #308.

- [x] dotnet.md
- [x] dsl.md - Deleted
- [x] javascript.md - Deleted
- [x] metadata.md
- [x] other-languages.md
- [x] swift.md
- [x] typescript.md

### Testing policies

- [x] will be closed by #366 - 01-intro.md
- [x] will be closed by #366 - 02-policy-authors.md
- [x] will be closed by #366 - 03-cluster-operators.md

## Site Styling

**Comment by jhkrug** [comment]
I think this first pass can be marked done.
